---
layout: archive
title: Research
permalink: /about/
layout: single
author_profile: true
header:
  overlay_image: /assets/Jellyfish_Banner2.png
gallery1:
  - url: /assets/photos/Museum_birds.JPG
    image_path: /assets/photos/Museum_birds.JPG
    alt: "AMNH starling collections"
    title: "AMNH starling collections"
  - url: /assets/photos/2016.Honours.3.jpg
    image_path: /assets/photos/2016.Honours.3.jpg
    alt: "Honours"
    title: "Honours"
  - url: /assets/photos/FA-IuxWUcA0Ugac.jpg
    image_path: /assets/photos/FA-IuxWUcA0Ugac.jpg
    alt: "Lizard co-author"
    title: "Lizard co-author"

gallery3:
  - url: /assets/photos/art_snake.jpg
    image_path: /assets/photos/art_snake.jpg
    alt: "This is a snake. It goes hissss."
    title: "This is a snake. It goes hissss."
  - url: /assets/photos/art_fish2.jpg
    image_path: /assets/photos/art_fish2.jpg
    alt: "This is a fish. It goes blub."
    title: "This is a fish. It goes blub."
  - url: /assets/photos/art_octo2.jpg
    image_path: /assets/photos/art_octo2.jpg
    alt: "This is an octopus. It predicts football match winners."
    title: "This is an octopus. It predicts football match winners."
---
I am an **evolutionary and conservation genomicist** investigating how genetic variants, from single nucleotide polymorphisms to large-scale structural changes, shapes adaptation across space and time. My research spans both invasive and native species, including plants and animals, to understand how **genetic diversity is distributed throughout the genome**, and how the **diverse and complex forms of genetic variation** underpin **population resilience or vulnerability** in the face of population disturbance or environmental change.

*** 
**Structural Variation in Ecoevolutionary genomics**
<div style="font-size: 0.8em;">
I am redefining how we study genetic differences between individuals. As whole genome resequencing becomes more accessible, a core focus of my research is how structural variants (SVs) and transposable elements can be better integrated into population genomics and conservation. Including these often-overlooked variants alongside SNPs offers a more complete view of genome-wide variation, which is crucial for tackling missing heritability, improving predictions for conservation breeding and translocation, and better understanding adaptive potential in uncertain futures.<br><br>

<strong>Recent work highlights:</strong>
<ul style="padding-left: 1.2em; padding-right: 1.2em; margin-top: 0.2em; list-style-type: none;">
  <li>→ Structural variation in eco-evolutionary genomics <a href="https://www.authorea.com/doi/full/10.22541/au.174853973.36642913/v1">Stuart et al., 2025, invited Mol Ecol – under review</a></li>
  <li>→ Heterozygous structural variants uniquely impact lifetime fitness <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17631">Stuart et al., 2025, Mol Ecol</a></li>
  <li>→ Population history shapes relative diversity levels between SVs and SNPs <a href="https://academic.oup.com/mbe/article/40/3/msad046/7052962">Stuart et al., 2023, MBE</a></li>
</ul>

</div>


![Evolutionary diagram](https://www.authorea.com/users/928924/articles/1300295/master/file/figures/image4/image4.png)

***
**Evolutionary Genomics of Rapid Adaptation**

My research explores how diverse genetic variants, from single nucleotide changes to complex structural variants and transposable elements, drive rapid adaptation in natural populations. By studying both invasive and native species across plants and animals, I integrate genomic, historical, and ecological data to uncover the mechanisms shaping evolutionary resilience and vulnerability.

These insights into rapid evolutionary processes directly inform conservation genomics. I apply cutting-edge genomic tools to real-world conservation challenges, especially in native Australian plants, aiming to maintain adaptive diversity through restoration and seed production. By translating evolutionary knowledge into practical strategies and developing accessible tools, my work bridges the gap between fundamental research and effective conservation management.


{% include gallery id="gallery1" layout="third" %}


***
**Beyond outliers: Using genetic outliers to investigate broader evolutionary hypotheses.**

Evolutionary genomic studies often produce a list of genetic sites flagged as under putative selection or associated with phenotype or environment data. 
<img style="float: right;" src="/assets/images/research_map3.png">
While generating a shortlist of biologically interesting variants is the first of many steps towards biologically validating these results, it is vital that we go beyond compilation of outlier loci and begin using this often end-result as a stepping stone to further scientific inquiry. To this end, I have been awarded funding from the <a href="https://ausevo.com/ECR_grants_2022/"> AES ERC Networking Grant Scheme</a> and <a href="https://genetics.org.au/">GSA</a> to develop workshop material for [genetic outlier analysis](https://github.com/katarinastuart/Ev1_SelectionMetaAnalysis). The ultimate goal of this project will be to develop cross species and cross study analyses (metanalyses) to interrogate a diverse list of species and population for broad trends that characterize genetic variants and regions undergoing contemporary selection. 

***
**The invasion genomics of the European starling**

The European starling *Sturnus vulgaris*, is one of the most widespread and iconic invasive avian species. However, despite their invasive range success, native starling populations are collapsing, with declines of up to 90% in the European countries they occupy. The global spread of this invasive species also provides unique opportunities to contrast genetic divergence across multiple invasive ranges, providing insight into the malleability of invasion regimes. I have worked across reduced representation sequencing (Stuart & Cardilini *et al* [2020](https://doi.org/10.1111/mec.15601)), whole genome resequencing (Hofmeister *et al* [2021](https://www.biorxiv.org/content/10.1101/2021.05.19.442026v1.abstract), Stuart *et al* [2023](https://doi.org/10.1093/molbev/msad046)), museomics (Stuart *et al* [2022a]( https://doi.org/10.1111/mec.16353)), and morphological data (Stuart *et al* [2022b](https://doi.org/10.3389/fgene.2022.824424)) to investigate the genetic underpinnings of the starling’s adaptive patterns across their global range, with a particular emphasis on the Australian population.

<img style="float: center;" src="/assets/images/AustralianStarling3.png" width="650" height="200">







