---
layout: archive
title: Research
permalink: /about/
layout: single
author_profile: true
header:
  overlay_image: /assets/Jellyfish_Banner2.png
gallery1:
  - url: /assets/photos/Museum_birds.JPG
    image_path: /assets/photos/Museum_birds.JPG
    alt: "AMNH starling collections"
    title: "AMNH starling collections"
  - url: /assets/photos/2016.Honours.3.jpg
    image_path: /assets/photos/2016.Honours.3.jpg
    alt: "Honours"
    title: "Honours"
  - url: /assets/photos/FA-IuxWUcA0Ugac.jpg
    image_path: /assets/photos/FA-IuxWUcA0Ugac.jpg
    alt: "Lizard co-author"
    title: "Lizard co-author"

gallery3:
  - url: /assets/photos/art_snake.jpg
    image_path: /assets/photos/art_snake.jpg
    alt: "This is a snake. It goes hissss."
    title: "This is a snake. It goes hissss."
  - url: /assets/photos/art_fish2.jpg
    image_path: /assets/photos/art_fish2.jpg
    alt: "This is a fish. It goes blub."
    title: "This is a fish. It goes blub."
  - url: /assets/photos/art_octo2.jpg
    image_path: /assets/photos/art_octo2.jpg
    alt: "This is an octopus. It predicts football match winners."
    title: "This is an octopus. It predicts football match winners."
---
I am an **evolutionary and conservation genomicist** investigating how genetic variation, from single nucleotide polymorphisms to large-scale structural changes, shapes adaptation across space and time. My research spans both invasive and native species, including plants and animals, to understand how **genetic diversity is distributed throughout the genome**, and how the **diverse and complex forms of genetic variation** underpin **population resilience or vulnerability** in the face of population disturbance or environmental change.

<hr style="width: 50px; border: 1px solid #006666; margin: 1em auto;">

**Structural Variants in Eco-evolutionary Genomics**
<div style="font-size: 0.8em;">
I am redefining how we study genetic differences between individuals. As whole genome resequencing becomes more accessible, a core focus of my research is integrating previously hidden components of genetic variation - structural variants (SVs) and transposable elements - into population genomics and conservation. Combining these often-overlooked variants with SNPs provides a more comprehensive view of genome-wide variation, which is essential for addressing missing heritability, improving predictions for conservation breeding and translocation, and better understanding adaptive potential in uncertain futures. <br><br>

<div style="display: flex; align-items: flex-start; gap: 0.5em;">

  <!-- Left side: text -->
  <div style="flex: 1; font-size: 0.9em;">
       <strong>My recent research highlights:</strong><br><br>
    <ul style="padding-left: 1.5em; padding-right: 1.5em; margin-top: 0.2em; list-style-type: none;">
      <li>❯ Structural variation in eco-evolutionary genomics (<a href="https://www.authorea.com/doi/full/10.22541/au.174853973.36642913/v1">Stuart et al., 2025</a>)</li><br>
      <li>❯ Heterozygous structural variants uniquely impact lifetime fitness (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17631">Stuart et al., 2024</a>)</li><br>
      <li>❯ Population history shapes relative diversity levels between SVs and SNPs (<a href="https://academic.oup.com/mbe/article/40/3/msad046/7052962">Stuart et al., 2023a</a>)</li><br>
    </ul>
  </div>

 <!-- Right side: image and caption -->
<div style="flex: 1; text-align: center;">
  <a href="https://www.authorea.com/doi/full/10.22541/au.174853973.36642913/v1" target="_blank" rel="noopener noreferrer" style="display: block; max-width: 600px; margin: 0 auto;">
    <img
      src="https://www.authorea.com/users/928924/articles/1300295/master/file/figures/image4/image4.png"
      alt="Subclassifications of different structural variants (SVs)"
      style="width: 100%; height: auto; display: block;"
    >
  </a>
  <div style="font-size: 0.9em; margin-top: 0; line-height: 1.2; max-width: 600px; margin-left: auto; margin-right: auto; text-align: center;">
    <strong>
      <a href="https://www.authorea.com/doi/full/10.22541/au.174853973.36642913/v1" target="_blank" rel="noopener noreferrer">Figure 3</a> | Subclassifications of different structural variants (SVs)
    </strong> that may need to be considered when conducting analysis, as they could impact pattern inference of different eco evolutionary processes.
  </div>
</div>


</div>

</div>

<hr style="width: 50px; border: 1px solid #006666; margin: 1em auto;">




**Evolutionary Genomics for Conservation & Restoration**
<div style="font-size: 0.8em;">
Understanding evolutionary processes and the distribution of genetic diversity is essential for effective management, conservation, and restoration. My research spans species with very small, isolated populations to those that are widely distributed, aiming to determine the best ways to preserve, safeguard, and sustainably utilise genetic resources. I focus on maintaining adaptive diversity through the application of evolutionary science to understand how, and when, genetic diversity or its absence influences population resilience. <br><br>

<div style="display: flex; align-items: flex-start; gap: 0.5em;">


<!-- Right side: image and caption -->
<div style="flex: 1; text-align: center;">
  <a href="https://www.nature.com/articles/s41437-024-00711-3" target="_blank" rel="noopener noreferrer" style="display: block; max-width: 600px; margin: 0 auto;">
    <img
      src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41437-024-00711-3/MediaObjects/41437_2024_711_Fig1_HTML.png?as=webp"
      alt="Linkage map of the hihi"
      style="width: 100%; height: auto; display: block;"
    >
  </a>
  <div style="font-size: 0.9em; margin-top: 0; line-height: 1.2; max-width: 600px; margin-left: auto; margin-right: auto; text-align: center;">
    <strong>
      <a href="https://www.nature.com/articles/s41437-024-00711-3" target="_blank" rel="noopener noreferrer">Figure 1</a> | Linkage map of the Hihi
    </strong> to better predict translocation and breeding outcomes of this entirely managed species.
  </div>
</div>

  <!-- Left side: text -->
  <div style="flex: 1; font-size: 0.9em;">
       <strong>My recent research highlights:</strong><br><br>
    <ul style="padding-left: 1.5em; padding-right: 1.5em; margin-top: 0.2em; list-style-type: none;">
      <li>❯ Shaping diversity and evolutionary resilience in native seed production areas (<a href="https://www.greeningaustralia.org.au/how-plant-dna-will-help-shape-the-future-of-native-seed-production-areas/">forthcoming</a>)</li><br>
      <li>❯ Managing inbreeding within endangered Hihi populations (<a href="https://www.nature.com/articles/s41437-024-00711-3">Tan et al., 2024</a>) (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17631">Stuart et al., 2024a</a>) (<a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.70024">Tan et al., 2025</a>) </li><br>
    </ul>
  </div>


</div>

</div>

<hr style="width: 50px; border: 1px solid #006666; margin: 1em auto;">




**Globally Invasive Common Starlings & Mynas**
<div style="font-size: 0.8em;">
My research investigates how diverse genetic variants, from single nucleotide changes to complex structural variants, facilitate rapid adaptation in the globally invasive common/European starling and common/Indian myna. This multidisciplinary work integrates genomic data with historical human records (such as newspapers and natural history collections), morphology, and ecological information to track the invasion footprint of these species worldwide. Together, these approaches help unravel how and why these two avian invaders have spread so successfully. <br><br>

<div style="display: flex; align-items: flex-start; gap: 0.5em;">

  <!-- Left side: text -->
  <div style="flex: 1; font-size: 0.9em;">
       <strong>My recent research highlights:</strong><br><br>
    <ul style="padding-left: 1.5em; padding-right: 1.5em; margin-top: 0.2em; list-style-type: none;">
      <li>❯ Historical starling DNA from museum collections reveal global and invasive genomic changes during the Anthropocene (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.16353">Stuart et al., 2022</a>)</li><br>
      <li>❯ Invasion driven change in a key domestication-associated gene shared between humans, dogs, sparrows, and invasive mynas (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17607">Atsawawaranunt et al., 2025</a>)</li><br>
      <li>❯ Historical newspaper records contextualize the starling invasion story told by DNA in New Zealand (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17579">Thompson et al., 2024</a>)</li><br>
      <li>❯ Climate variability underlies morphological changes in Australia’s invasive starling populations (<a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.824424/full">Stuart et al., 2023b</a>)</li><br>
    </ul>
  </div>

<!-- Right side: image and caption -->
<div style="flex: 1; display: flex; flex-direction: column; align-items: center; justify-content: center; text-align: center;">
  <a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1010456/full" 
     target="_blank" 
     rel="noopener noreferrer" 
     style="display: block; max-width: 600px;">
    <img
      src="/assets/images/AustralianStarling_v2.png"
      alt="Starling invasion in Australia"
      style="width: 65%; height: auto; display: block; margin: 0 auto;"
    >
  </a>
  <div style="font-size: 0.9em; margin-top: 0; line-height: 1.2; max-width: 600px; text-align: center;">
    <strong>
      <a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1010456/full" 
         target="_blank" 
         rel="noopener noreferrer">Figure 1</a> | Population structure and evolutionary trends of the common starling in Australia
    </strong> including the genomic location of loci under selection, and associations between genetic, morphological, and environmental variability.
  </div>
</div>



</div>

</div>

<hr style="width: 50px; border: 1px solid #006666; margin: 1em auto;">



**Population Management for Invasive Species**
<div style="font-size: 0.8em;">
I use population genomics and evolutionary principles to directly inform conservation strategies for managing invasive species. By applying cutting-edge genomic tools to real-world challenges, I translate evolutionary insights into practical management approaches and develop accessible resources that bridge the gap between fundamental research and effective conservation action. <br><br>

<div style="display: flex; align-items: flex-start; gap: 0.5em;">


<!-- Right side: image and caption -->
<div style="flex: 1; display: flex; flex-direction: column; align-items: center; justify-content: center; text-align: center;">
  <a href="https://academic.oup.com/gbe/article/16/1/evad230/7477876?login=true" target="_blank" rel="noopener noreferrer" style="display: block; max-width: 600px; margin: 0 auto;">
    <img
      src="/assets/images/Museum_genomics_update3.png"
      alt="Linkage map of the hihi"
      style="width: 70%; height: auto; display: block; margin: 0 auto;"
    >
  </a>
  <div style="font-size: 0.9em; margin-top: 0; line-height: 1.2; max-width: 600px; margin-left: auto; margin-right: auto; text-align: center;">
    <strong>
      <a href="https://academic.oup.com/gbe/article/16/1/evad230/7477876?login=true" target="_blank" rel="noopener noreferrer">Figure 3</a> | The historical context of invasions
    </strong> helps to (A) contextualize shifts in genetic diversity within both native and introduced ranges and (C) model range shifts relative to present-day invasive species distributions & diversity.
  </div>
</div>


  <!-- Left side: text -->
  <div style="flex: 1; font-size: 0.9em;">
       <strong>My recent research highlights:</strong><br><br>
    <ul style="padding-left: 1.5em; padding-right: 1.5em; margin-top: 0.2em; list-style-type: none;">
      <li>❯ Genomic applications in invasion biology (<a href="https://academic.oup.com/gbe/article/16/1/evad230/7477876">McGaughran & Dhami et al. 2024</a>) (<a href="https://books.google.com.au/books?hl=en&lr=&id=34XmEAAAQBAJ&oi=fnd&pg=PA286&dq=info:3CbH7nO0hDAJ:scholar.google.com&ots=upS-UrtlYs&sig=2Qi5byw3YHLMNM0tewDISpqZfPg&redir_esc=y#v=onepage&q&f=false">Stuart et al. 2023c</a>)</li><br>
      <li>❯ Genomic-informed deer management in the Illawarra region (<a href="https://www.publish.csiro.au/WR/WR22128">Li-Williams & Stuart et al., 2023</a>)  </li><br>
    </ul>
  </div>


</div>

</div>

<hr style="width: 50px; border: 1px solid #006666; margin: 1em auto;">


**Genomic diveristy & variation: Future research directions**
<div style="font-size: 0.8em;">
❯ Kangaroo paws, hybird zones, and strucutral variants. <br>
❯ Detecting population genetic variation in eDNA. <br>
❯ Population genomics of Australian native plants.<br>
  
<div style="display: flex; align-items: flex-start; gap: 0.5em;">









