---
layout: archive
title: Research
permalink: /research/
layout: single
author_profile: true
header:
  overlay_image: /assets/images/banner1.png
---

<style>
  hr { 
    display: block;
    margin-before: 0.5em;
    margin-after: 0.5em;
    margin-start: auto;
    margin-end: auto;
    overflow: hidden;
    border-style: inset;
    border-width: 1px;
}
  </style>

My research interests are in **evolutionary genomics** and exploring the proximate mechanisms that facilitate local adaptation. Most of my research is conducted on **introduced species**, as successful invasives provide an important opportunity to observe rapid evolution within a species as it responds to a new ecosystem. Understanding **rapid adaptation** within invasive populations has applications in many scientific fields, including but not limited to pest control, conservation, organism response to climate change, and evolutionary mechanisms. Most of my research is focused on the invasive starling population in Australia and focuses on identifying adaptive genetic shifts post invasion. 

My projects and interests cover a range of fields, including <b>genomics</b>, <b>bioinformatics</b>, <b>ecological statistics</b>, and <b>field ecology</b>.

***
**The invasion genomics of the European starling**

The European starling *Sturnus vulgaris*, is one of the most widespread and iconic invasive avian species. However, despite their invasive range success, native starling populations are collapsing, with declines of up to 90% in the European countries they occupy. The global spread of this invasive species also provides unique opportunities to contrast genetic divergence across multiple invasive ranges, providing insight into the malleability of invasion regimes. I have worked across reduced representation sequencing (Stuart & Cardilini *et al* [2020](https://doi.org/10.1111/mec.15601)), whole genome resequencing (Hofmeister *et al* [2021](https://www.biorxiv.org/content/10.1101/2021.05.19.442026v1.abstract), Stuart *et al* [2023](https://doi.org/10.1093/molbev/msad046)), museomics (Stuart *et al* [2022a]( https://doi.org/10.1111/mec.16353)), and morphological data (Stuart *et al* [2022b](https://doi.org/10.3389/fgene.2022.824424)) to investigate the genetic underpinnings of the starlingâ€™s adaptive patterns across their global range, with a particular emphasis on the Australian population.

<img style="float: center;" src="/assets/images/AustralianStarling3.png" width="650" height="200">


***
**Avian genomes**

Interpreting results surround adaptive selection is in part based on the genomic tools available for the analysis. 
<img style="float: left;" src="/assets/images/research_genome_trans2.png" width="200" height="200">
To improve our ability to map future genetic read data, assembling high quality genomes is essential. I have assembled the genome of *Sturnus vulgaris* (Stuart & Edwards *et al* [2022](https://doi.org/10.1111/1755-0998.13679)) and *Acridotheres tristis* (forthcoming) was part of broader work on these species. I am generally interested in genome assembly, and the affects assembly choices may have on our ability to examine population structure, and identify adaptive changes. 

***
**Beyond outliers: Using genetic outliers to investigate broader evolutionary hypotheses.**

Evolutionary genomic studies often produce a list of genetic sites flagged as under putative selection or associated with phenotype or environment data. 
<img style="float: right;" src="/assets/images/research_map3.png">
While generating a shortlist of biologically interesting variants is the first of many steps towards biologically validating these results, it is vital that we go beyond compilation of outlier loci and begin using this often end-result as a stepping stone to further scientific inquiry. To this end, I have been awarded funding from the <a href="https://ausevo.com/ECR_grants_2022/"> AES ERC Networking Grant Scheme</a> and <a href="https://genetics.org.au/">GSA</a> to develop workshop material for [genetic outlier analysis](https://github.com/katarinastuart/Ev1_SelectionMetaAnalysis). The ultimate goal of this project will be to develop cross species and cross study analyses (metanalyses) to interrogate a diverse list of species and population for broad trends that characterize genetic variants and regions undergoing contemporary selection. 

*** 
**A holistic view of genetic variation**

With whole genome resequencing data become more available, my current research directions are exploring how structural variants and transposable elements respond to population bottlenecks. This includes the theory that stress induced greater transposable element translocation, and thus may be a source of novel genetic material in invasive populations.
