---
layout: archive
title: Research
permalink: /research/
layout: single
author_profile: true
header:
  overlay_image: /assets/images/banner1.png
---

<style>
  hr { 
    display: block;
    margin-before: 0.5em;
    margin-after: 0.5em;
    margin-start: auto;
    margin-end: auto;
    overflow: hidden;
    border-style: inset;
    border-width: 1px;
}
  </style>

My research interests are in **evolutionary genomics** and exploring the proximate mechanisms that facilitate local adaptation. Most of my research is conducted on **introduced species**, as successful invasives provide an important opportunity to observe rapid evolution within a species as it responds to a new ecosystem. Understanding **rapid adaptation** within invasive populations has applications in many scientific fields, including but not limited to pest control, conservation, organism response to climate change, and evolutionary mechanisms. Most of my research is focused on the invasive starling population in Australia and focuses on identifying adaptive genetic shifts post invasion. 

My projects and interests cover a range of fields, including <b>genomics</b>, <b>bioinformatics</b>, <b>ecological statistics</b>, and <b>field ecology</b>.

***
**The invasion genomics of the European starling**

The European starling *Sturnus vulgaris*, is one of the most widespread and iconic invasive avian species. However, despite their invasive range success, native starling populations are collapsing, with declines of up to 90% in the European countries they occupy. The global spread of this invasive species also provides unique opportunities to contrast genetic divergence across multiple invasive ranges, providing insight into the malleability of invasion regimes. I have worked across reduced representation sequencing (Stuart & Cardilini *et al* [2020](https://doi.org/10.1111/mec.15601)), whole genome resequencing (Hofmeister *et al* [2021](https://www.biorxiv.org/content/10.1101/2021.05.19.442026v1.abstract), Stuart *et al* [2023](https://doi.org/10.1093/molbev/msad046)), museomics (Stuart *et al* [2022a]( https://doi.org/10.1111/mec.16353)), and morphological data (Stuart *et al* [2022b](https://doi.org/10.3389/fgene.2022.824424)) to investigate the genetic underpinnings of the starlingâ€™s adaptive patterns across their global range, with a particular emphasis on the Australian population.

<img style="float: center;" src="/assets/images/AustralianStarling.png" width="650" height="200">


***

Interpreting results surround adaptive selection is in part based on the genomic tools available for the analysis. 
<img style="float: left;" src="/assets/images/research_genome_trans2.png" width="200" height="200">
To improve our ability to map future genetic read data and understand which genomic regions variants fall within for <i>S. vulgaris</i>, I have drafted a <b>genome assembly </b>for this species. This genome project also contains the first species specific transcript driven gene models for annotation. I am generally interested in genome assembly, and the affects assembly choices may have on our ability to examine population structure, and identify adaptive changes.

***

<img style="float: right;" src="/assets/images/research_map3.png">

The <b>phenotypic changes</b> an invasive species undergo during invasion are a result of complex interactions between environment and genotype. I am interested in exploring different ways of analysing large phenotypic data sets, and combining these with information about the environment and/or genetics.
