---
layout: archive
title: Teaching & Vignettes
permalink: /vignettes/
layout: single
author_profile: true
header:
  overlay_image: /assets/images/banner1.png

feature_row:
  - image_path: /assets/vignettes/vignette_preview_sv3b.jpg
    alt: "Genome Plots"
    title: "Genome Plots"
    excerpt: "How to use **Circlize** in R to plot pretty much anything around your genome."
    url: "https://github.com/katarinastuart/Sv3_StarlingGenome"
    btn_label: "Read More"
    btn_class: "btn--inverse"
  - image_path: /assets/vignettes/vignette_preview_sv4b.jpg
    alt: "SNP variant calling"
    title: "SNP variant calling"
    excerpt: "Calling SNPs from **raw DArTseq data** using a few mapping and variant calling softwares."
    url: "https://github.com/katarinastuart/Sv4_HistoricalStarlings"
    btn_label: "Read More"
    btn_class: "btn--inverse"
  - image_path: /assets/vignettes/vignette_preview_sv2b.jpg
    alt: "Distribution Map"
    title: "Distribution Map"
    excerpt: "Retrieving data from eBird and using it to plot a **species distribution map**."
    url: "https://github.com/katarinastuart/Sv2_StarlingLitReview"
    btn_label: "Read More"
    btn_class: "btn--inverse"
  - image_path: /assets/vignettes/vignette_preview_sv6b.jpg
    alt: "Morphology & Genetics"
    title: "Morphology & Genetics"
    excerpt: "Some association tests between genetic and morphological data."
    url: "https://github.com/katarinastuart/Sv6_StarlingEcogeographical"
    btn_label: "Read More"
    btn_class: "btn--inverse"
  - image_path: /assets/vignettes/vignette_preview_sv5c.jpg
    alt: "SV Calling"
    title: "SV Calling"
    excerpt: "Calling **structural variants** from short read whole genome resequencing data."
    url: "https://github.com/katarinastuart/Sv5_StarlingWGS"
    btn_label: "Read More"
    btn_class: "btn--inverse"

---

**Teaching & Training**

I have taught across undergraduate and postgraduate levels in both Australia and New Zealand, delivering content in evolutionary biology, genetics, and genomics. My teaching spans lectures, tutorials, and laboratory demonstrations, with a focus on engaging students across diverse skill levels—from foundational molecular biology to advanced bioinformatics.

At the undergraduate level, I have contributed to courses such as Genes, Genomes and Evolution, Principles of Molecular Biology, and Evolution of Genes, Populations and Species. I have led population genetics labs, taught core genomics principles, and introduced students to evolutionary bioinformatics.

I am also committed to postgraduate training and capacity building in genomic data analysis. I developed and lead a Genetic Outlier Analysis workshop series, delivered across New Zealand and Australia, with over 150 participants to date. These workshops provide hands-on training in PCAdapt, FST, BayeScan, and BayPass, including support for attendees working with their own datasets.

Additionally, I have contributed as a facilitator for coding workshops with Genomics Aotearoa and served as a teaching assistant at the Data Intensive Biology Summer Institute (ANGUS) at UC Davis, training participants in large-scale sequencing data analysis.

I am passionate about inclusive teaching and have contributed to Indigenous science education programs, developing and delivering biology content as part of UNSW’s Science and Engineering Indigenous Preparatory Program.

***
**Full workshop material are available online for:**

- **[Redundancy Analysis](https://github.com/katarinastuart/EnvComputing_RDA)**: Ecology and multivariate statistics covering **redundancy analysis** of cane toad phenotype data.
- **[Genetic Outlier Analysis](https://github.com/katarinastuart/Ev1_SelectionMetaAnalysis)**: Covering the basics of **PCAdapt**, **Fst**, **BayeScan**, and **BayPass**, and some discussion around selection analysis on genomic data. Another version of this workshop is available through [GenomicsAotearoa](https://genomicsaotearoa.github.io/Outlier_Analysis_Workshop), which has been updated to run on New Zealand eScience Infrastructure (NeSI).

In addition to this, I like running discussion groups about how to construct thesis chapters and manuscripts. Some of the notes on this **[can be found here](https://github.com/katarinastuart/Constructing_a_Manuscript/tree/main)**.

***
Below is a collection of vignettes I have made over the years. My general goal is to try create one vignette per first author paper, in it capturing some aspect of the data processing or analysis that I had to learn for the project. My aim in creating these is is threefold:<br>
1. increase my research repeatability and transparancy
2. provide a starting point for other researches who need help with the code
3. reflect on new program/techniques learnt duing projects and note areas of improvment for the future

I do not imagine these vignettes are free from errors, however if you spot any major ones or would just like general assistance with anything feel free to contact me. 

{% include feature_row %}
